tests/test_seeds.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import pandas as pd
+import statsmodels.formula.api as sm
+from wildboottest.wildboottest import wildboottest
+import numpy as np
+
+@pytest.fixture
+def data():
+    N = 100
+    k = 2
+    G= 20
+    X = np.random.normal(0, 1, N * k).reshape((N,k))
+    beta = np.random.normal(0,1,k)
+    beta[0] = 0.1
+    u = np.random.normal(0,1,N)
+    y = 1 + X @ beta + u
+    cluster = np.random.choice(list(range(0,G)), N)
+    bootcluster = cluster
+    R = np.zeros(k)
+    R[0] = 1
+    B = 999
+    
+    return X, y, cluster, bootcluster, R, B
+
+def test_results_from_same_seed():
+    
+    df = pd.read_csv("https://raw.github.com/vincentarelbundock/Rdatasets/master/csv/sandwich/PetersenCL.csv")
```

Comment:
This is likely fine for now, but I in general try to avoid loading anyting from the web in tests, as doing so adds an one more point of potential failure. This is super bad in R, as CRAN would kick you off within two weeks :D

Reply:
Dropped the www dependency